src/explore-education-statistics-admin/src/pages/release/data/ReleaseDataFilePage.tsx

```diff
@@ -76,14 +76,17 @@ export default function ReleaseDataFilePage({
             <FormProvider
               initialValues={{ title: dataFile.title }}
               validationSchema={Yup.object<FormValues>({
-                title: Yup.string().required('Enter a title'),
+                title: Yup.string()
+                  .required('Enter a title')
+                  .max(120, 'Subject title must be 120 characters or less'),
```

Comment:
Nit pick, but using magic number of `120` might lead to a small bug down the line if we want to update the `maxLength`.  We have "120" appear in 3 places here, we might think about using a `const titleMaxLength = 120` and using/interpolating it into the validation/messages